### PR TITLE
Issue #281 - Use the most-recent version, not the most-used version for the crash dash

### PR DESCRIPTION
**Pull request by chutten**
The crash dash currently displays the most-used version per channel in an
effort to maximize the crash rate denominator "kuh".

This isn't terribly helpful as, once a new release lands, we cease to really
care about the stability of the old one.

So track the release schedule using https://product-details.mozilla.org and
only use data from the most recent release.

Issue #281

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/telemetry-dashboard/282)

### CODE REVIEW

**Comment by chutten** [comment]
Live preview can be found here: https://chutten.github.io/telemetry-dashboard/crashes/

**Comment by chutten** [comment]
This needs some tweaking. It needs to auto-select a day from the latest version instead of waiting for usage to increase enough to match the previous version's use, and the crash plot can probably just stop truncating on the x-axis.

File: crashes/latest-versions.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+  let addVersion = function (version, add) {
+    if (!add) {
+      return version;
+    }
+    let major = version.split('.')[0];
+    return (window.parseInt(major, 10) + add) + ".0"
+  };
+
+	var formatYYYYmmdd = function(d) {
```
Comment (fbertsch) [inline crashes/latest-versions.js]: Indentation?

File: crashes/index.html [hunk 2]
```diff
@@ -61,6 +62,9 @@
 var gData = undefined;
 var gMap = {};
 
+// When populated, this is a map activity_date -> channel -> version
+var gLatestVersions = getLatestVersions();
```
Comment (fbertsch) [inline crashes/index.html]: With limit=10 (at least, I *think* that is what is causing the problem), some options for "analyzed date" aren't showing (well, they show, but they don't update the table). Check the earliest date (2016-11-08) for an example of this. Gets a "TypeError: gMap[weekAgoDate] is undefined".

**Review by fbertsch — COMMENTED** [review]
1. The y-axis is *way* below the max y-value, is this expected?

2. The % next to each number doesn't seem very relevant anymore (sways wildly between 10^-2 and 10^4, depending on how far after the release). Maybe it's best to compare the X days after the previous release for that channel?

File: crashes/index.html (hunk 2 above)
Comment (chutten) [reply crashes/index.html]: The first seven dates don't work as they rely on data being present a week before. I suppose I can make that safe, but it's always been that way. I've filed #284 for this.

**Review by chutten — COMMENTED** [review]
1. The y axis is locked so that crash spikes don't reduce the rest of the graph to a flat line near 0.

2. I've struggled with the usefulness of the %ge stuff. It's a real pain, but if I remove them users may misunderstand the veracity of the dash's claims. As I can't guarantee that the data for multiple releases will be the same (or even present in the query results) I don't think I can use "release + x days" as a metric. (especially if a release is delayed).

This leaves me unfortunately uncertain about what to do with them.

**Review by fbertsch — APPROVED** [review]
LGTM

**Pull request merged**